### ISSUE TITLE
/usr/bin/env: 'bash\r': No such file or directory

### ISSUE THREAD

**Issue by OrlandoST** [yiisoft/yii2-docker#79]
I am getting the error when trying to up container:

/usr/bin/env: 'bash\r': No such file or directory
ERROR: Service 'php' failed to build: The command '/bin/sh -c curl -sS https://getcomposer.org/installer | php --
  --filename=composer.phar         --install-dir=/usr/local/bin &&     composer clear-cache' returned a non-zero code: 1

**Comment by schmunk42** [comment]
Hmm, are you on Windows?

Something like this, maybe? ... https://help.github.com/en/articles/configuring-git-to-handle-line-endings#platform-windows

**Comment by hypotheses** [comment]
I had a similar problem and landed here from a Google search. 

In my case, I used Github Desktop to pull the repository to my computer running Windows10 with Insider Program to set up `wsl2` subsystem for Ubuntu18.04. This seems like the recommended set up to run the new Docker Desktop with wsl2 support.

At first, running `docker-compose up -d` complete the build of all related containers without any problem. However, the web service running `apache` kept restarting. 

The log file showed this same error `/usr/bin/env bash\r`: No such file or directory. 

I tried changing `/usr/bin/env bash` to `/bin/bash` but this caused more complications with `pecl install xdebug` giving an error that "xdebug verion 2.9.4 does not have REST dependency information available". 

My final working solution was deleting the repository downloaded via Github desktop for windows. 
Get the repository with `git clone` via Ubuntu Subsystem instead.

One more step that I had to do was deleting all the previously build container images. Continue building with `docker-compose up -d` from the previously failed `pecl install xdebug` still resulted in the same error message. 

After rebuilding everything again from scratch, all the services now work properly.
